weave-js/src/components/PagePanelComponents/Home/Browse3/context.tsx

```diff
@@ -449,6 +463,26 @@ export const browse3ContextGen = (
         leaderboardName ? `/${leaderboardName}` : ''
       }${edit ? '?edit=true' : ''}`;
     },
+    compareCallsUri: (
+      entityName: string,
+      projectName: string,
+      callIds: string[]
+    ) => {
+      const params = callIds
+        .map(id => 'call=' + encodeURIComponent(id))
+        .join('&');
+      return `${projectRoot(entityName, projectName)}/compare?${params}`;
+    },
+    compareObjectsUri: (
```

Comment:
Are Ops supported as objects?

Reply:
I see by playing with it that they are not - seem like we could support that in the future

Reply:
Yes, that would be a good future enhancement to make things more consistent. Comparing calls was already scope creep I couldn't resist. :-)